src/components/InvestmentPercentages/Bonds/BondsPercentages.tsx

```diff
@@ -5,6 +5,7 @@ import { Cell, Pie, PieChart, Tooltip } from 'recharts'
 
```

Comment:
Can we try abstracting all these Files: `<>Percentages.tsx` into a single component? They roughly do the same thing